src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -31,6 +31,14 @@ public static class ILoggerExtensions
             + ContextProperties.DependencyTracking.IsSuccessful + "} - Context: {@"
             + ContextProperties.EventTracking.EventContext + "})";
 
+        private const string ServiceBusDependencyFormat =
+            MessagePrefixes.Dependency + " Azure Service Bus {" 
```

Comment:
Do we use this approach, or do we create a converter like others?

Reply:
See https://github.com/arcus-azure/arcus.observability/pull/76/files?file-filters%5B%5D=.cs#diff-75d51c6e532953a0a10af8789970abeeR25

Reply:
I think having it would be ideal because I'd like to add `EntityType` as a context property which will be `Queue`, `Topic` or `Unknown` (when not specified, see below)

Reply:
1. I like the way we now reuse existing custom dependency functionality, actually. So the distinction is made at client level but not internally. Especially if there would be no extra functionality.
2. Yes, that would be good! I was wondering actually which dependency tracking properties we should use for this, what their values should be, ...

Reply:
2. Done.